src/renderers/WebGLRenderer.js

```diff
@@ -2901,7 +2905,7 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 	}
 
-	function setCubeTexture ( texture, slot ) {
+	function setTextureCube ( texture, slot ) {
```

Comment:
I see. Well, the function names didn't really reflect that difference and there is no documentation about this. But I understand that and will revert these changes.